scripts/packaging/createnugetpackages.ps1

```diff
@@ -137,15 +137,22 @@ try
 
         nuget pack $_.FullName -OutputDirectory $OutputDirectory -Properties $props -Exclude *.nuspec.meta
         
+        # ===
+        # https://github.com/microsoft/MixedRealityToolkit-Unity/issues/6276 states that not having a version on the assemblies in our NuGet packages
+        # is an issue... commenting the localVersion update
+        # ===
+
         # To make debugging the MRTK NuGet packages locally much easier automatically create new packages with version 0.0.0 and then
         # restore them to the machine NuGet feed. To test changes to the packages developers can run this script and then change their
         # project to consume version 0.0.0 and restore. Because the package is in the machine global feed it will resolve properly.
-        $localVersion = '0.0.0'
+        # $localVersion = '0.0.0'
```

Comment:
Remove?

Reply:
i'd like to leave it in until we are sure we no longer need / want it